### PR TITLE
Allow boolean attributes to be used in `get_* -filter` expressions without comparison

### PR DESCRIPTION
**Pull request by akashlevy**
See title.

### CODE REVIEW

**Comment by akashlevy** [comment]
Hey @jjcherry56, wanted to bump this one. Any interest in upstreaming?

File: test/get_filter.tcl [hunk 1]
```diff
@@ -12,6 +12,16 @@ puts "get_clocks"
 report_object_full_names [get_clocks -filter is_virtual==0 *]
 puts "get_clocks 2"
 report_object_full_names [get_clocks -filter is_virtual==1 *]
+puts "get_clocks 3"
+report_object_full_names [get_clocks -filter is_virtual *]
```
Comment (jjcherry56) [inline test/get_filter.tcl:16]: I put use {} to print what is being tested so it is visible in the report:
puts {[get_clocks -filter is_virtual *]}

**Comment by jjcherry56** [comment]
It makes sense. I'd like a comment or two in the code and a note in the filter section of the doc.

**Comment by akashlevy** [comment]
Thanks for the feedback. I've made the requested changes

**Pull request merged**